### PR TITLE
New Cheat Sheet: Russian language

### PR DESCRIPTION
**Pull request by mayank7**
New Cheat Sheet: Russian language

IA Page: https://duck.co/ia/view/russian_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Russian Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/russian_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Russian Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/russian_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Russian Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/russian_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Russian Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/russian_cheat_sheet_mobile.png)

**Comment by jdorweiler** [comment]
@mayank7 I'm going to tag all your language cheat sheets as needing review from a language expert.  We would like to get some sign off from a second source for each of these cheat sheets.

File: share/goodie/cheat_sheets/json/language/russian.json [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+            "trn": "vi Ochin’ dabrY",
+            "val": "Вы очень добры"
+         }, {
+            "key": "Where are you from?",
+            "trn": "Vy atkUda?",
+            "val": "Вы откуда?"
+         }, {
+            "key": "I'm from (the U.S/ Russia)",
+            "trn": "iz (ShtAtaf/ Rasii)",
+            "val": "из (Штатов/ России)"
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/language/russian.json]: This should be:
trn: `ya iz shtAtof / rOsii`
val: `Я из Штатов/ России`

**Pull request closed without merging**